### PR TITLE
show monthly top topics on 404 page

### PR DESCRIPTION
**Pull request by techAPJ**
cc @SamSaffron @coding-horror

### CODE REVIEW

**Comment by coding-horror** [comment]
basically a one-liner so I think it's OK unless you have concerns @SamSaffron

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -1232,6 +1232,7 @@ def post_user
           guardian = Guardian.new(user)
           guardian.stubs(:ensure_can_edit!).with(user).raises(Discourse::InvalidAccess.new)
           Guardian.stubs(new: guardian).with(user)
+          Guardian.stubs(new: guardian).with(nil)
```
Comment (ZogStriP) [inline spec/controllers/users_controller_spec.rb]: Are you stubbing it with `user` or `nil`?

Comment (techAPJ) [inline spec/controllers/users_controller_spec.rb]: `nil`, so on 404 page monthly top topics will be loaded assuming that the user is not logged in.

Comment (ZogStriP) [inline spec/controllers/users_controller_spec.rb]: Then the previous line isn't necessary, isn't it?

Comment (techAPJ) [inline spec/controllers/users_controller_spec.rb]: Okay, I updated the spec via https://github.com/discourse/discourse/commit/6df5b38b54d265fef0e6a9749ff89839db898374

**Pull request merged**